poetry.lock

```diff
@@ -612,141 +633,86 @@ version = "1.11.0"
 description = "library with cross-python path, ini-parsing, io, code, log facilities"
 optional = false
 python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
+groups = ["dev"]
 files = [
     {file = "py-1.11.0-py2.py3-none-any.whl", hash = "sha256:607c53218732647dff4acdfcd50cb62615cedf612e72d1724fb1a0cc6405b378"},
     {file = "py-1.11.0.tar.gz", hash = "sha256:51c75c4126074b472f746a24399ad32f6053d1b34b68d2fa41e558e6f4a98719"},
 ]
 
 [[package]]
 name = "pydantic"
-version = "2.9.2"
-description = "Data validation using Python type hints"
+version = "1.10.24"
+description = "Data validation and settings management using python type hints"
 optional = false
-python-versions = ">=3.8"
+python-versions = ">=3.7"
```

Comment:
weird...not sure why pydantic went back many versions....

Reply:
seems it is related to the `packaging` version update [here](https://github.com/uc-cdis/argo-wrapper/pull/164/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R19), which was needed to get rid of the build errors. Looks like packaging had moved to v24.1 which is incompatible. In short: looks like `packaging` needs to be v23 (and then pydantic can be >2) or v25 (and then pydantic can only be < 2).